laravel/app/Http/routes.php

```diff
@@ -81,12 +81,15 @@
 Route::get('/schedule/{schedule}/delete', 'ScheduleController@deleteForm');
 Route::post('/schedule/{schedule}/delete', 'ScheduleController@delete');
 
-
-// Slot routes
-Route::get('/slot/{slot}/view', 'SlotController@view');
-Route::post('/slot/{slot}/take', 'SlotController@take');
-Route::post('/slot/{slot}/release', 'SlotController@release');
-
+//Routes for Admin
+Route::group(['middleware' => ['auth','admin']], function()
```

Comment:
The admin middleware should only be used around the slot edit route. Adding it to all of these slot routes prevents normal users from signing up for shifts.